### ISSUE TITLE
[BUG] cannot create jail with routable IPv6

### ISSUE THREAD

**Issue by Duponin** [BastilleBSD/bastille#186]
**Describe the bug**
When I create a jail with a routable IPv6 address, an error is thrown (status code `1`).

**Bastille and FreeBSD version**
```
0.5.20191128
12.1-RELEASE-p3
12.1-RELEASE-p3
12.1-RELEASE-p3
```

**How did you install bastille? (port/pkg/git)**
Bastille is installed with `pkg`.

**Steps to reproduce?**
Create a jail with an IPv6 address starting with a `2001:`
Ex:
```
sudo bastille create amazing-jail 12.1-RELEASE 2001:470[...] gif0
```

**Expected behavior**
Bastille should ask me if I really to use a non private IP address.
That behavior should be the same with IPv4 and IPv6 (IPv4 jails can have public IP addresses).

**Additional context**
I use IPv6 to get rid of IPv4 limitations.
I don't have NAT, jails are on the WAN interface.
I tried with a routable IPv4 and no error was thrown.

**Comment by Duponin** [comment]
Okay I dived into code to inspect.
In version `0.5.20191128` which is available with FreeBSD 12.1 IPv6 is not supported.

I installed it from sources on master branch to have IPv6 support.
Now it accepts to create jail with IPv6.

**Comment by cedwards** [comment]
Thank you.

Will close this now that ports/pkg is updated past 0.5 and includes IPv6.
